tests/Stubs/Externals/ORM/ManagerRegistryStub.php

```diff
@@ -0,0 +1,116 @@
+<?php
```

Comment:
Have you run phpunit inspections over this? Empty stubs is usually one of the biggest fixes I have to do because the interface annotates a return type (doesn't look like it does here but I dunno) and therefore something is expected to be returned even if it's empty.

Reply:
This is me being lazy. Didnt run inspections, will flesh them out with proper stubs (boo, the mocks were also me being lazy ;))

Reply:
This doesnt report any problems when inspected.